package.json

```diff
@@ -17,9 +17,9 @@
     "lint": "prettier --check \"{lib,test}/**/*.ts\" && tslint --format stylish \"{lib,test}/**/*.ts\"",
     "format": "prettier --loglevel warn --write '{lib,test}/**/*.ts' && tslint --fix --format stylish '{lib,test}/**/*.ts'",
     "test": "npm run lint && npm run unit-test",
-    "test-jest": "jest --bail --logHeapUsage",
+    "test-jest": "jest --ci --maxWorkers=3 --logHeapUsage",
```

Comment:
Do we know why `3` works best here? :)

Reply:
Took it from other repos we have. Found smth online :)
https://www.peterbe.com/plog/ideal-number-of-workers-in-jest-maxWorkers

Reply:
Ah thats good to know about! TIL. Thanks!